README.md

```diff
@@ -9,18 +9,24 @@ These files are in SMIRKS Native Open Force Field (SMIRNOFF) format.
 Details about this new format are documented in our recent publication ([doi:10.1021/acs.jctc.8b00640](https://www.doi.org/10.1021/acs.jctc.8b00640) or [bioRxiv](https://doi.org/10.1101/286542)), and the most recent documentation of the specification can be found in the [Open Force Field Toolkit documentation](https://open-forcefield-toolkit.readthedocs.io/en/latest/smirnoff.html).
 You can parameterize small molecules with SMIRNOFF using the
 `ForceField` class in the [openforcefield toolkit](https://github.com/openforcefield/openforcefield)
-for simulations with [OpenMM](http://openmm.org/).
+for simulations with [OpenMM](http://openmm.org/), or other simulation formats supported by [ParmEd](http://parmed.github.io/ParmEd/html/index.html).
 
 Usage examples can be found in the [openforcefield repository](https://github.com/openforcefield/openforcefield/tree/master/examples).
 
+Each force field is currently available in two forms -- Both with and without bond constraints to hydrogen. 
+The default version of each force field has bond constraints to hydrogens, and is suitable for long-timestep simulations.  
```

Comment:
Not sure "long-timestep simulations" is the right phrase; maybe "typical molecular dynamics simulations with constrained bonds to hydrogen"? I think we want people to default to thinking, "this is the one I should be using if I'm doing MD" not "this is the one I should use if I'm using long timesteps".

Reply:
Changed in 7812e48